src/app/retrospective/retrospect-task-modal/retrospect-task-modal.component.html

```diff
@@ -50,6 +50,7 @@ <h6 class="subtitle">
             [stopEditingWhenGridLosesFocus]="true"
             [overlayLoadingTemplate]="overlayLoadingTemplate"
             [overlayNoRowsTemplate]="overlayNoRowsTemplate"
+            [suppressScrollOnNewData]="true"
```

Comment:
check if these properties could be set inside `gridOptions`. If yes, carefully move it into the `setGridOptions` function

Reply:
Also, check every other ag-grid for similar pattern